### PR TITLE
test: split fixtures for browser test  

### PR DESCRIPTION
**Pull request by hi-ogawa**
### Description

Base branch: https://github.com/vitest-dev/vitest/pull/4867 (new change is only the last commit)

Testing to see if splitting the custom base test would affect test failure on CI.
I also rewrote an assertion to be based on stdout/stderr, so it might be easier to debug.

I'll retry CI for a few times by force-pushing the same commit.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [x] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

### Tests
- [ ] Run the tests with `pnpm test:ci`.

### Documentation
- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [x] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

File: test/browser/specs/custom-base.test.mjs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import assert from 'node:assert'
+import test from 'node:test'
+import { execa } from 'execa'
+
+test('custom-base', async () => {
+  const result = await execa('npx', [
+    'vitest',
+    'run',
```
Comment (AriPerkkio) [inline test/browser/specs/custom-base.test.mjs:8]: Was this using `npx` before? Signal terminals are buggy when scripts are launched via `npx`. I can't find the link for the issue or older references for these issues right now. Consider using `startVitest` instead.

Comment (AriPerkkio) [reply test/browser/specs/custom-base.test.mjs:8]: `pnpm exec` should also work correctly.

Comment (hi-ogawa) [reply test/browser/specs/custom-base.test.mjs:8]: Currently browser mode testing is written with `execa` + `npx vitest`, so I was keeping the same structure from:

https://github.com/vitest-dev/vitest/blob/c946b1c841c04f27f026c2b10b7b8f16ac12ceab/test/browser/specs/run-vitest.mjs#L12

The test runner used here is `node --test`, so test utils like `runVitest` are not available and `runner.test.mjs` would probably be easier to written with `execa`, so I'll keep that there for now.

This small test `custom-base.test.mjs` should be fine with `startVitest` API, so I'll rewrite that. Thanks for the suggestion!

**Comment by hi-ogawa** [comment]
Closing this in favor of the other one
- https://github.com/vitest-dev/vitest/pull/4879

**Pull request closed without merging**